src/file-upload/default-formatters.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { formatDateTime } from '../internal/utils/date-time';
+
+export function formatFileSize(size: number): string {
+  const KB = 1000;
+  const MB = 1000 ** 2;
+  return size < MB ? `${(size / KB).toFixed(2)} KB` : `${(size / MB).toFixed(2)} MB`;
```

Comment:
Can we include GB and TB as well? That way we can cover the common file sizes

Reply:
I think GB is not unlikely at all. For example, if you want to [upload an existing virtual machine into EC2](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html#import-vm-image) or large data sets for machine learning, this will be multiple GB in many cases.
While TB sizes are currently unlikely because of the long upload time on most consumer connections, I think it is good to be on the safe side here by supporting "one step larger" than we expect, especially since it requires very little code. We will then not have to think about this function again for many years.

Reply:
Is it possible to have a fake file object? Can we use this component not for uploads from the local system directly, but from some remote endpoint to another?

In any way, adding extra units does not cost us much, so why not to add it regardless